app/views/shared/_head.html.haml

```diff
@@ -15,6 +15,8 @@
   I18n.defaultLocale = "#{I18n.default_locale}";
   I18n.locale = "#{I18n.locale}";
 
+  TimeZones = #{ActiveSupport::TimeZone.all.each_with_object({}) { |v,h| h[v.name] = v.formatted_offset }.to_json};
```

Comment:
Maybe instead of doing this translation on each page load, we should set up the array in an initializer.